engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -1568,4 +1568,6 @@ UPDATE
 SET
   usage_type = 22
 WHERE
-  usage_type = 24 AND usage_display like '% io write';
\ No newline at end of file
+  usage_type = 24 AND usage_display like '% io write';
+
+CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.guest_os', 'display', 'tinyint(1) DEFAULT ''1'' COMMENT ''should this guest_os be shown to the end user'' ');
```

Comment:
@DaanHoogland 
as we discussed, add this to 4.16.0->4.16.1->4.17.0->4.17.1->4.17.2 as well ?

Reply:
no, I checked (and will test) the guestosmapper is not yet used in those.